wlm-ui/src/MainPage/MainPage.tsx

```diff
@@ -11,13 +13,26 @@ const MainPage = () => {
         dispatch(signout());
     };
 
+    const onClickRefreshChannelList = async () => {
+        dispatch(fetch());
+    };
+
     return (
         <div>
             <button
                 onClick={onClickSignout}
             >
                 Sign out
             </button>
+            <div>
+                <h1>Channel list</h1>
+                <button
+                    onClick={onClickRefreshChannelList}
+                >
```

Comment:
Following my suggestion, this should be packed in a single line.

Reply:
Then, please apply it!